### PR TITLE
[feat] - Create Guides Tab

### LINKED ISSUE #102: Create guides tab in resources page
https://linear.app/trap-garden/issue/TG-68/create-the-guides-tab-in-resources-page

### PR DESCRIPTION
**Pull request by torinnayak1**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Created the guides tab with pdf previews for each of the types of gardens. The user can click on the header or the preview itself and the pdf will be opened in a new tab. Also added the icons to the Resource Page header. Moved some functions from onboarding into helpers for easier reuse of code.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

Screenshot 2025-04-13 at 5 48 28 PM

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Please review the PDFSmallPreview files and the changes in the resources page. Everything else is either adding an icon, or moving functions to helpers. The last commit is just adding the icons, so please review the first commit for the most functional changes

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @kylezryr

### CODE REVIEW

File: app/resources/page.tsx [hunk 1]
```diff
@@ -134,6 +155,33 @@ export default function Resources() {
         <H1 $color="white" $fontWeight={500}>
           Resource Page
         </H1>
+        <div style={{ position: 'absolute', top: '3.5rem', left: '0rem' }}>
```
Comment (kylezryr) [inline app/resources/page.tsx]: could we make these divs a component inside of this same file and then render it here? you can call it something like `IconsDisplay`

File: app/resources/styles.ts [hunk 2]
```diff
@@ -35,3 +35,14 @@ export const PageContainer = styled.div`
   minheight: '100vh';
   backgroundcolor: 'white';
 `;
+
+export const GuidesGrid = styled.div`
+  margin: 1.5rem;
```
Comment (kylezryr) [inline app/resources/styles.ts]: instead of a margin here can we wrap the whole grid display in a container, then add padding of 1.5rem? sometimes margins cause the screen to be buggy if there's no padding in the outermost container

File: components/PDFSmallPreview/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+export default function PDFSmallPreview(selectedGardenType: UserTypeEnum) {
+  const url = getPDFUrl(selectedGardenType);
+  return (
+    <Flex
+      $direction="column"
+      $justify="center"
+      $gap=".75rem"
+      $align="center"
+      $w="10.5rem"
```
Comment (kylezryr) [inline components/PDFSmallPreview/index.tsx:23]: can we also add a `$minH$ property to this Flex, because when loading the PDF the components are small then get bigger once the image loads, as in the clip. check Figma for what the height should be!

https://github.com/user-attachments/assets/d2245675-0f48-4e5f-815b-2f4380ba7c2c

**Review by kylezryr — CHANGES_REQUESTED** [review]
really well done! not much comments to add, just a couple of changes to make. we'll need to come back to this for desktop view in the future but great job for now! i rebased the branch so please 

1. git checkout main and git pull origin main 
2. git checkout branch name 
3. git pull

and sort out any merge conflicts, it should all be accept current changes i think
